src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -410,7 +410,9 @@ void LottieBuilder::appendRect(Shape* shape, Point& pos, Point& size, float r, b
 
     if (modifier) {
         auto& path = SHAPE(temp)->rs.path;
-        modifier->modifyPath(path.cmds.data, path.cmds.count, path.pts.data, path.pts.count, nullptr, SHAPE(shape)->rs.path);
+        //sharp rectangles need to be handled separately - the close command really completes the shape
```

Comment:
Not good. You undermines the purpose of introducing the ShapeModifier.

Reply:
solved